supportedBackends/aws/src/test/scala/cromwell/backend/impl/aws/AwsBatchInitializationActorSpec.scala

```diff
@@ -187,28 +185,26 @@ class AwsBatchInitializationActorSpec extends TestKitSuite with AnyFlatSpecLike
 
   behavior of "AwsBatchInitializationActor"
 
-  // it should "log a warning message when there are unsupported runtime attributes" taggedAs IntegrationTest in {
-  //   AwsAuthModeSpec.assumeHasApplicationDefaultCredentials()
-  //
-  //   within(Timeout) {
-  //     val workflowDescriptor = buildWdlWorkflowDescriptor(HelloWorld,
-  //       runtime = """runtime { docker: "ubuntu/latest" test: true }""")
-  //     val backend = getAwsBatchBackend(workflowDescriptor, workflowDescriptor.callable.taskCallNodes,
-  //       defaultBackendConfig)
-  //     val eventPattern =
-  //       "Key/s [test] is/are not supported by backend. Unsupported attributes will not be part of job executions."
-  //     EventFilter.warning(pattern = escapePattern(eventPattern), occurrences = 1) intercept {
-  //       backend ! Initialize
-  //     }
-  //     expectMsgPF() {
-  //       case InitializationSuccess(_) => //Docker entry is present.
-  //       case InitializationFailed(failure) => fail(s"InitializationSuccess was expected but got $failure")
-  //     }
-  //   }
-  // }
-
-  // Depends on https://github.com/broadinstitute/cromwell/issues/2606
-  it should "return InitializationFailed when docker runtime attribute key is not present" taggedAs PostWomTest ignore {
+  it should "log a warning message when there are unsupported runtime attributes" taggedAs IntegrationTest in {
```

Comment:
I don't understand why this test was commented out, but if it works now, :shipit: